src/adapter/src/catalog/config.rs

```diff
@@ -19,6 +19,9 @@ use mz_storage::types::hosts::StorageHostResourceAllocation;
 
 use crate::catalog::storage;
 
+/// The default interval to collect storage usage information
+pub const DEFAULT_STORAGE_METRIC_INTERVAL_SECONDS: &str = "3600";
```

Comment:
It can :sweat_smile:  It was easier to pass a string to the [arg handler](https://github.com/MaterializeInc/materialize/pull/13718/files#diff-18db540d7592188856fb9af92ebb418ad154a7b8c04e88c798f33a99b530c0a6R364) but to your point, I'm not sure how configurable the interval needs to be. I might remove those pieces after all